server/routes/index.js

```diff
@@ -23,9 +29,11 @@ router.post('/login', login);
 router.post('/brand', isAuth, isAdmin, postBrand);
 router.post('/type', isAuth, isAdmin, postType);
 
+router.put('/update/car/:carId', isAuth, isAdmin, updateCarControllers);
```

Comment:
i think its better like that
```suggestion
router.put('/cars/:carId', isAuth, isAdmin, updateCarControllers);
```